### PR TITLE
Make JSON for website

### PR DESCRIPTION
**Pull request by grst**
Concatenate all `meta.yaml` files into a single json array

- [x] deploy only on *commit* to main branch

### CODE REVIEW

File: .github/workflows/make-json.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Make JSON
+on:
+    pull_request:
+        branches: [main]
```
Comment (ivirshup) [inline .github/workflows/make-json.yml:4]: Shouldn't this only build for things that have been merged?

```suggestion
```

**Comment by grst** [comment]
@ivirshup / @gtca 

This is ready, feel free to approve&merge.

File: .github/workflows/make-json.yml (hunk 1 above)
Comment (grst) [reply .github/workflows/make-json.yml:4]: It should always build, but only deploy once it's merged. 
This is done with an `if` in the deploy step.

Comment (ivirshup) [reply .github/workflows/make-json.yml:4]: Ah, cool, just saw that commit

I wrote the review last night, but only realized I hadn't posted it once I saw your ping 😅

**Review by ivirshup — APPROVED** [review]

**Comment by ivirshup** [comment]
Whats with the failing deployment? Not obvious to me where that's coming from

**Comment by ivirshup** [comment]
Ah, I think that should be fixed once this is merged?

**Comment by grst** [comment]
> Whats with the failing deployment? Not obvious to me where that's coming from

There's now an alternative way of deploying github pages (without pushing to `gh-pages` branch). I couldn't make it work, so I reverted back to the 'old' way.

**Comment by grst** [comment]
In any case, it seems to have worked, as we have the full list now: 
https://scverse.org/ecosystem-packages/ecosystem.json

**Pull request merged**